hkmc2Benchmarks/src/test/bench/LazySpreads/LazyArrayTest.mls

```diff
@@ -1,25 +1,24 @@
 :js
-:runtime ../mlscript-compile/lazy-spread-runtimes/RuntimeLazyArray.mjs
-
-//│ Overriding runtime: ../mlscript-compile/lazy-spread-runtimes/RuntimeLazyArray.mjs
 
 import "../../../../../hkmc2/shared/src/test/mlscript-compile/LazyArray.mls"
 import "../mlscript-compile/Benchmark.mls"
+import "../../../../../hkmc2/shared/src/test/mlscript-compile/Runtime.mls"
 
 open LazyArray
 open Benchmark
+open Runtime
+
+set Runtime.Tuple.lazySlice = (xs, i, j) => LazyArray.slice(i, j)(xs)
+set Runtime.Tuple.lazyConcat = (...args) => LazyArray.concat(...args)
+set Runtime.Tuple.split = LazyArray.__split
```

Comment:
Don't forget to do the same for all tests, even if the overriding doesn't change the semantics.

Reply:
This PR only contains LazyArrayTest, as the FingerTree stuff are in a separate PR. I will update the benchmark suite, after they're both merged.

Nevertheless, since the benchmarks are run from lambda functions added to the suite, and this runtime overriding isn't being timed, would we expect it to affect the benchmark numbers? (I might be misunderstanding what you mean)

Reply:
If you compare a version of the benchmarked program that uses a non-overridden Runtime with one that does, the results might be biased. Since overriding module members may have a performance effect.

> This PR only contains LazyArrayTest, as the FingerTree stuff are in a separate PR. I will update the benchmark suite, after they're both merged.

Ok.